src/extension.ts

```diff
@@ -11,107 +12,55 @@ interface Params {
     file: string | null
 }
 
-interface SentryProject {
-    projectId: string
-    lineMatches: RegExp[]
-}
-
 const DECORATION_TYPE = sourcegraph.app.createDecorationType()
 const SETTINGSCONFIG = resolveSettings(sourcegraph.configuration.get<Settings>().value)
+const SENTRYORGANIZATION = SETTINGSCONFIG['sentry.organization']
+
+const COMMON_ERRORLOG_PATTERNS = [
+    /throw new Error+\(['"]([^'"]+)['"]\)/gi,
+    /console\.[^'"`]+\(['"`]([^'"`]+)['"`]\)/gi,
```

Comment:
This appears to match strings like `console.("foo")`. Don't you want to match `/console\.(log|error|info|warn)/`?

Reply:
I don't seem to be getting the same matches as you:
![image](https://user-images.githubusercontent.com/9110008/55370010-cc967a80-54ad-11e9-9a87-92322d902be7.png)

Reply:
But you are right: It will match wrong code like `console.l("test)` e.g. so maybe I should safeguard it with `(log|error|info|warn)`.